src/win/snprintf.c

```diff
@@ -0,0 +1,49 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
I also thought that, but since the code here was just moved around I wasn't sure if I should point it out. What do you suggest? "Copyright the libuv project contributors" ?